_haikus/mobility_increase.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Mobility Increase
+author: Arpit Shrotriya
+---
+
+Better Algorithms for Path Calculations<br>
+ML models to create the Congestion control<br>
+Make life better.<br>
```

Comment:
We need some changes on the haiku portion of this.

Haikus follow a syllabic format, so each line should be a certain number of syllables.

Your first line should be 5 syllables, the second line 7 syllables, and the last line 5 syllables.

You can check to make sure you have the right number of syllables by using a tool such as http://www.haikusyllablecounter.com/.

Once you've updated the haiku to follow this format, we can merge it!